packages/p2p/src/stores/general-store.js

```diff
@@ -740,7 +740,7 @@ export default class GeneralStore extends BaseStore {
             return;
         }
 
-        const { p2p_order_list, p2p_order_info } = order_response;
+        const { p2p_order_list, p2p_order_info } = order_response || {};
```

Comment:
```suggestion
        const { p2p_order_list = [], p2p_order_info = {} } = order_response;
```
If you want to add fallback this is the correct way. De-structuring `{ p2p_order_list, p2p_order_info }` from an empty object `{}` will still result in both these variables as `undefined`